test/deep_stacks.cc

```diff
@@ -0,0 +1,29 @@
+#include <chrono>
+#include <iostream>
+#include <string>
+
+template <int N> std::string compute() {
```

Comment:
Compiler will probably fully inline all the function and the recursive calls (and perhaps even figure out directly the final value).